extensions/micrometer/runtime/src/main/doc/usage.adoc

```diff
@@ -0,0 +1,37 @@
+This extension leverages https://quarkus.io/guides/micrometer[Quarkus Micrometer]. Quarkus supports a variety of Micrometer metric registry implementations.
+
+Your application should declare one of the following dependencies, depending on the monitoring solution you want to work with.
+
+[cols="50,.^50]
+|===
+|Monitoring System | Dependency
+
+| Azure
+
+| `micrometer-registry-azure-monitor`
```

Comment:
What are the groupIds of these deps?

Reply:
Whoops! I forgot that part. I'll add it.

Reply:
Latest commit adds the group ids